inventory_management_system_api/core/exceptions.py

```diff
@@ -21,21 +21,21 @@ class NonLeafCategoryError(Exception):
     """
 
 
-class DuplicateCatalogueItemPropertyNameError(Exception):
+class DuplicateCategoryPropertyNameError(Exception):
     """
-    Catalogue category is attempted to be created with duplicate catalogue item property names.
+    Catalogue category is attempted to be created with duplicate property names.
     """
 
 
-class InvalidCatalogueItemPropertyTypeError(Exception):
+class InvalidPropertyTypeError(Exception):
     """
-    The type of the provided value does not match the expected type of the catalogue item property.
+    The type of the provided value does not match the expected type of the property.
     """
 
 
-class MissingMandatoryCatalogueItemProperty(Exception):
+class MissingMandatoryProperty(Exception):
     """
-    A mandatory catalogue item property is missing when a catalogue item is attempted to be created.
+    A mandatory property is missing when a catalogue item is attempted to be created.
```

Comment:
I think this one is also raised when an item is attempted to be created so should we update the docstring to mention item?